filament/include/filament/Engine.h

```diff
@@ -239,6 +240,13 @@ class UTILS_PUBLIC Engine {
          * This value does not affect the application's memory usage.
          */
         uint32_t perFrameCommandsSizeMB = FILAMENT_PER_FRAME_COMMANDS_SIZE_IN_MB;
+
+        /**
+         * Whether to use an irradiance map for the environment lighting.
+         *
+         * NOTE: This is not currently supported. Always set to false.
+         */
+        bool useIrradianceMap = false;
```

Comment:
I'm confused why this belongs under `Engine::Config`?

Reply:
Sounds good. Seems like this belongs under `IndirectLight` or something, but maybe I'm not understanding its purpose.

Reply:
It's a precomputed texture for the diffuser of the ibl. Currently, we use the roughness=1 of the prefiltered cubemap, but this is technically both incorrect and low quality (which only matters for certain scenes).

To use the irradiance map, we'd need an extra sampler, first of all, but also, we'd need to either compute it from roughness=0, or, get it from the user (and we don't have an API for that). Precomputing it at runtime, would take a few hundred ms probably.

So it's one of those feature that we should really have, but is not critical.